andela/learning-map-front
# #150216138 Create Contribution Detail Modal

#### What does this PR do?
- Creates a modal to display the details of a contribution

#### Description of Task to be completed?
When a contribution is clicked on the contributions page, the user should be able to see details of the contribution.

#### How should this be manually tested?
- Start the application
- Log in with your andela email
- When you're successfully logged in, you should see the different contributions
- Click on any of the contributions

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#150002165](https://www.pivotaltracker.com/story/show/150002165)

#### Screenshots (if appropriate)
![screen shot 2017-08-24 at 12 42 49 pm](https://user-images.githubusercontent.com/25608335/29665086-5b627400-88ca-11e7-852d-35d071c0fb5a.png)
![screen shot 2017-08-24 at 12 42 40 pm](https://user-images.githubusercontent.com/25608335/29665097-6cda43ac-88ca-11e7-8182-86ca51aa7fe6.png)


#### Questions:
N/A
